### DISCUSSION TITLE
On merging some pull requests

### DISCUSSION THREAD

**Discussion by kmk3** [netblue30/firejail#5479] [General]
So there are quite a few pull requests that have been opened for a while.

Most of the following ones are relatively simple and all of them have had no
changes (and no request for changes) for more than two weeks:

* #5459 (no changes since 2022-11-07)
* #5449 (no changes since 2022-11-03)
* #5442 (no changes since 2022-10-30)
* #5429 (no changes since 2022-10-22)
* #5352 (no changes since 2022-09-26)

This one seems simple enough and has had no changes for over a week:

* #5473

Thoughts on merging them?

The more commits that are made to master in the meantime, the longer (and
potentially wider) the merge lines are going to be, which makes the git history
harder to read.  Merging the existing PRs would help with that on at least
branches that are created/rebased after the merges.

Also, there may be branches in development that relate to or that are blocked
by existing PRs (I have some, though the relevant PR is not in the above
lists).

Lastly, because I think it's helpful when projects are responsive.

**Comment by Fred-Barclay** [comment]
Good call @kmk3 ! I'll review these and merge the ones I'm familiar/comfortable with - let's get them all in/any changes needed made!

**Comment by ghost** [comment]
Thanks for the list. #5352 and #5459 are merged now.

**Comment by kmk3** [comment]
All merged!

Thanks for the feedback, reviews and merges.
